### PR TITLE
Icons in a namespace for better usage and typing

### PR DESCRIPTION
**Pull request by strouhaljan**
---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: libs/sdk-ui-kit/src/Icon/Icon.tsx [hunk 1]
```diff
@@ -1,51 +1,38 @@
...
+    ExternalLink,
+    Hyperlink,
+    Undo,
+    Home,
+    BurgerMenu,
+    Rows,
+    DragHandle,
+    Interaction,
+    AttributeFilter,
+};
```
Comment (scavnickyj) [inline libs/sdk-ui-kit/src/Icon/Icon.tsx:38]: Add newline please

File: libs/sdk-ui-tests/stories/visual-regression/kit/Icon/Icon.tsx [hunk 2]
```diff
@@ -8,31 +8,70 @@ import { withScreenshot } from "../../../_infra/backstopWrapper";
...
-            <IconWrapper name="Pdf" color="#0f0" />
-            <IconWrapper name="ExternalLink" color="#00f" />
-            <IconWrapper name="Hyperlink" color="#f00" />
-            <IconWrapper name="Undo" color="#0f0" />
-            <IconWrapper name="Home" color="#00f" />
-            <IconWrapper name="DragHandle" color="#f00" />
-            <IconWrapper name="AttributeFilter" color="#0f0" />
-            <IconWrapper name="Interaction" color="#00f" />
+            <IconWrapper name="Refresh">
+                <Icon.Refresh color={"#f00"} />
```
Comment (scavnickyj) [inline libs/sdk-ui-tests/stories/visual-regression/kit/Icon/Icon.tsx:28]: No need for brackets in these color props, right?

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3122/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4776/ : FAILURE in 6m 10s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3125/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4778/ : SUCCESS in 12m 38s

**Comment by strouhaljan** [comment]
extended test

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/790/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3127/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/790/ : SUCCESS in 18m 38s

**Comment by strouhaljan** [comment]
extended test

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/792/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3132/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/792/ : UNSTABLE in 18m 18s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3131/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4784/ : SUCCESS in 16m 48s

**Comment by strouhaljan** [comment]
extended test - filter "Icon"

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3136/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/794/ : ABORTED in 1h 00m 13s

**Review by scavnickyj — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1224

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/936/ : SUCCESS in 54s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1204/ : SUCCESS in 22s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4789/ : SUCCESS in 12m 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/320/ : SUCCESS in 8m 26s (non-voting)

**Pull request merged**